pymc3/gp/gp.py

```diff
@@ -136,8 +136,8 @@ def sample_gp(trace, gp, X_values, samples=None, obs_noise=True, model=None, ran
     # Posterior covariance
     S_post = S_zz - tt.dot(tt.dot(S_xz.T, S_inv), S_xz)
 
-    gp_post = MvNormal.dist(m_post, S_post, shape=Z.shape[0])
-    
+    gp_post = MvNormal.dist(m_post, S_post + jitter * tt.eye(S_post.shape[0]), shape=Z.shape[0])
```

Comment:
I like `chol_const`.  Want to have it as an arg like `jitter` is now?  Or do you like

    n = S_post.shape[0]
    chol_const * tt.nlinalg.trace(S_post)/n * tt.eye(n)

better?